Make last-monday-of-the-month releases a fixed deadline (#4057)
PostHog/posthog.com: contents/handbook/engineering/release-new-version.md

showTitle: true
---

We release a new version on the last Monday of every month.

> This consistency is important, as it means our community and our customers can look forward to new features at a predictable pace. There will always be more work that we want to do and if we get in a habit of pushing deadlines out, we'll push them further and further.

Reviewer: ```suggestion
This consistency is important, as it means our community and our customers can look forward to new features at a predictable pace. There will always be more work that we want to do and if we get in a habit of pushing deadlines out, we'll push them further and further. If a feature isn't ready by the release date, cut it or release it behind a feature flag.
```

> Sprints do not line up with releases, so a release might fall in the middle of the sprint, for which we need to plan accordingly.

Reviewer: ```suggestion
Sprints do not line up with releases, so a release might fall in the middle of the sprint, for which we need to plan accordingly.  If a feature wouldn't be ready by the release date, cut down the scope, remove it or release it behind a feature flag.
```

> Sprints do not line up with releases, so a release might fall in the middle of the sprint, for which we need to plan accordingly.  If a feature wouldn't be ready by the release date, cut down the scope, remove it or release it behind a feature flag.

Reviewer 2: ```suggestion
Sprints do not line up with releases, so a release might fall in the middle of the sprint, for which we need to plan accordingly. If a feature won't be ready by the release date then cut down the scope, remove it, or release it behind a feature flag.
```
